### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by finnp** [hoodiehq/hood.ie#137]
Hey :),

there are a few things that could be improved about the a11y of your website. Have a look at this audit here: http://wave.webaim.org/report#/http%3A%2F%2Fhood.ie . Especially the default contrast of your top nav bar slightly fails AA, so you might want to make that font bigger or darken the orange a bit.

I would love to PR some changes, but your repo is like 140 Mb big and needs forever for me to clone. Maybe there are some binary files you can remove from the repo history?

Best,
Finn

**Comment by verpixelt** [comment]
Hey @finnp thanks for you offer <3. I'll look into the issue with the repository file size. It's the blog media, of two years, which bloats everything up.

**Comment by janl** [comment]
The file size is due to media on the blog, sorry. We should sign up for this: https://github.com/blog/1986-announcing-git-large-file-storage-lfs

**Comment by janl** [comment]
I signed us up, let’s see what happens :)

**Comment by verpixelt** [comment]
That's a good idea but we should clean up this folders anyway. There are 4 versions of every image created by Wordpress, which we used before.

I will have a look into it sometime this week.

**Comment by varjmes** [comment]
Hey @finnp, thank you so much for bringing this to our attention.
I'm sorry that we haven't responded in a while but please know we are thinking about this. I recently became an official contributor to Hood.ie and my almost-complete focus is making hood.ie accessible. Tackling the main website is certainly at the top of my list and I will be using this PR and my own auditing to put together a list of things that must be improved to create an awesomely accessible experience :)

**Comment by finnp** [comment]
@Charlotteis That sounds really great! Glad to see someone take care of this.

**Comment by varjmes** [comment]
@janl @verpixelt We now have git-lfs enabled on this repository, as Jan tells me. Here is how to use it: https://github.com/github/git-lfs#getting-started. I am trying to do it myself, but am having some installation problems, so if someone else could also take a look: that'd be great.

**Comment by varjmes** [comment]
I got it working. However, it seems that git-lfs will not work retrospectively. That is, when we start tracking the files we want, it will only turn them into pointers when we add new ones. So, we need to find a solution to de-bigify (a very technical word) our repository as it currently is. I believe @janl has some thoughts.

My question is, what file extensions would we like to track for git-lfs?

**Comment by varjmes** [comment]
I've started to talk about large file storage over here: #173
